rpc/block_svc_routines.c

```diff
@@ -2591,7 +2593,7 @@ getTpgObj(char *block, MetaInfo *info, blockGenConfigCli *blk, char *portal, int
   uuid_generate(uuid);
   uuid_unparse(uuid, alias);
   snprintf(alias_10, 11, "%.10s", alias+24);
-  json_object_object_add(tpg_lun_obj, "alias", GB_JSON_OBJ_TO_STR(alias_10));
+  json_object_object_add(tpg_lun_obj, "alias", GB_JSON_OBJ_TO_STR(alias_10[0]?alias_10:NULL));
```

Comment:
Are this and the following changes really needed here ?

Reply:
These are to avoid gcc and rpmbuild warnings. Are you saying we should make this as a separate patch/PR ?

Reply:
I test the old code, couldn't see any warning info of this. I thought the alias_10[0] couldn't be '\0' because of the last line of the code would fill something in it.

Reply:
Nice. Could you try:

$ make clean
$ make CFLAGS="-Wall"

Thanks!

Reply:
Cool.